clusterloader2/api/validation.go

```diff
@@ -55,9 +55,17 @@ func (v *ConfigValidator) Validate() *errors.ErrorList {
 		allErrs = append(allErrs, v.validateTuningSet(c.TuningSets[i], field.NewPath("tuningSets").Index(i))...)
 	}
 
+	// validate steps
+	isStepEmpty := len(c.Steps) == 0
+	if isStepEmpty  {
+		allErrs = append(allErrs, field.Invalid(field.NewPath("steps"),0,"cannot be empty"))
+
+	}
+	else {
```

Comment:
You don't need this else. If there are no steps, this "for" loop below will do nothing.